modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -342,7 +346,8 @@ def claimant_birth_date_is_in_the_past
       return if claimant.birth_date.blank?
 
       unless self.class.past?(claimant.birth_date)
-        add_error("Claimant birth date isn't in the past: #{claimant.birth_date_string}")
+        add_error("Claimant birth date isn't in the past: #{claimant.birth_date_string}",
+                  title: "Birth date isn't in the past", pointer: '/data/attributes/claimant', code: 144)
```

Comment:
@cilestin code: 148 ?, which is 'invalid value'

Reply:
@stiehlrod No I think your use of code 144 is good, I more meant my struggle is with how can we best indicate that there's header issues (of _any_ sort) when `pointer` doesn't seem to have a way to express that the header is where the issue lies.

For example, when the pointer is `/data/attributes/representative/name` being in the wrong data type, the pointer shows directly where the issue in the data occurred, but since there is no "path" to the header data that was provided, so far we've resorted to using `/` for the pointer and using `meta` as the way to indicate which header actually had the issue.

I don't think it's a bad way to do it, just open to alternatives and happy to discuss them if you come up with any :)